app/controllers/admin/roles_controller.rb

```diff
@@ -7,7 +7,7 @@ class RolesController < BaseController
     def index
       authorize :user_role, :index?
 
-      @roles = UserRole.order(position: :desc).page(params[:page])
+      @roles = UserRole.order(position: :desc).includes(:users).page(params[:page])
```

Comment:
There is a `role.users.count` in the template, but will this kind of preload be effective for it? Also, the `@everyone` role is part of this too, and I think it might load up all users in that preload (but the template does not trigger a count for it).

Reply:
Yes - it's coming from that ... though upon further review I think this is a false positive. Going to close this PR with a note.